### PR TITLE
Subscription response

### PR DESCRIPTION
**Pull request by MattChilders92**

### CODE REVIEW

File: src/runtime/callbacks.js [hunk 1]
```diff
@@ -144,50 +143,41 @@ export class Callbacks {
    * @param {function(!Promise<!../api/subscribe-response.SubscribeResponse>)} callback
    */
   setOnSubscribeResponse(callback) {
-    this.setCallback_(CallbackId.SUBSCRIBE_RESPONSE, callback);
+    console./*OK*/warn(`[swg.js:setOnSubscribeResponse]: This method has been deprecated, please swith usages to 'setOnPaymentResponse'`);
```
Comment (alin04) [inline src/runtime/callbacks.js]: typo.  s/swith/switch

Comment (MattChilders92) [reply src/runtime/callbacks.js]: Good catch, thanks

File: src/runtime/callbacks.js [hunk 2]
```diff
@@ -144,50 +144,45 @@ export class Callbacks {
    * @param {function(!Promise<!../api/subscribe-response.SubscribeResponse>)} callback
    */
   setOnSubscribeResponse(callback) {
-    this.setCallback_(CallbackId.SUBSCRIBE_RESPONSE, callback);
+    console./*OK*/ warn(
```
Comment (jpettitt) [inline src/runtime/callbacks.js]: add a warn method to log.js and call that rather than putting the console.warn here (ditto below)

Comment (MattChilders92) [reply src/runtime/callbacks.js]: Done

**Pull request merged**